tests/test_sunsch.py

```diff
@@ -27,13 +27,20 @@ def readonly_datadir():
         os.chdir(cwd)
 
 
-def test_main(tmpdir, caplog, mocker):
-    """Test command line sunsch, loading a yaml file"""
-
+@pytest.fixture
+def testdata(tmpdir):
     tmpdir.chdir()
+    cwd = os.getcwd()
```

Comment:
This could probably also be performed with pathlib

Reply:
Fixed!

Reply:
> I see 6 lines using os in test_sunch.py. Are you not able to lift the os requirement entirely?

`os.chdir()` has no pathlib-replacement.